scripts/bot-build.slurm

```diff
@@ -32,12 +32,20 @@ else
     exit 1
 fi
 echo "bot/build.sh finished"
-CHECK_RESULT_SCRIPT=bot/check-result.sh
-if [ -f ${CHECK_RESULT_SCRIPT} ]; then
-    echo "${CHECK_RESULT_SCRIPT} script found in '${PWD}', so running it!"
-    ${CHECK_RESULT_SCRIPT}
+CHECK_BUILD_SCRIPT=bot/check-build.sh
+
+# temporarily also still consider bot/check-result.sh script as a fallback
+CHECK_RESULT_SCRIPT=bot/check-results.sh
```

Comment:
I think that is unnecessary. If the bot would be in use in many places for many different repositories, it might make sense.

Reply:
Well, we do need the corresponding change in `compatibility-layer` and `software-layer` (and in all branches...) first then?

I wouldn't mind having this safeguard in place for a couple of days to ease the transition, I don't see any downsides here.
I can open another PR right after this one to strip out this fallback mechanism again, so we don't forget about doing so...

Reply:
removed fallback in [4b013d3](https://github.com/EESSI/eessi-bot-software-layer/pull/189/commits/4b013d3c976425a81b7264bdcd0a09ec05cc9f37)